qa/rpc-tests/p2p-addrv2-relay.py

```
@@ -0,0 +1,97 @@
+        # Have receiver advertise addrv2 support
+        print("Sending sendaddrv2 message")
+        msg_receiver.connection.send_message(msg_sendaddrv2())
+        msg_receiver.sync_with_ping()
+
+        print("Testing that sendaddrv2 message is accepted")
+        # If we got here without disconnect, sendaddrv2 was accepted
+        print("✓ Node accepted sendaddrv2 message successfully")
+        
+        # Test that regular addr messages still work
+        print("Sending regular addr messages (IPv4)")
+        addr_msg = msg_addr()
```

Ruff (0.14.10)

`msg_addr` may be undefined, or defined from star imports

(F405)